mmv1/products/networkconnectivity/Spoke.yaml

```diff
@@ -114,6 +141,13 @@ properties:
         min_version: ga
         required: true
         immutable: true
+      - !ruby/object:Api::Type::Array
+        name: includeImportRanges
+        description: |
+          IP ranges allowed to be included during import from hub (does not control transit connectivity).
+          The only allowed value for now is "ALL_IPV4_RANGES".
+        item_type: Api::Type::String
+        default_from_api: true
```

Comment:
Thanks for adding the test coverage for the different `linked_x` blocks!

A consequence of making this field default from the API is that users will have trouble setting this field to an empty value once the field is set. Is there any scenario where a user might want/need to set this to an empty array?

Reply:
Fair point - I added incase customers were already managing this by manually mutating the resource due to no TF support, but there are scenarios where customers may want to set this to an empty array. Will update.

Reply:
Thanks!